src/linux/mod.rs

```diff
@@ -27,6 +27,19 @@ type Phdr = libc::Elf32_Phdr;
 type Phdr = libc::Elf64_Phdr;
 
 const NT_GNU_BUILD_ID: u32 = 3;
+const PT_NULL: u32 = 0;
```

Comment:
Sadly libc crate does not add these constants to android, we need to manually copy them here for now.